### PR TITLE
Added JaCoCo

### PR DESCRIPTION
**Pull request by LiangZhang21**
Added JaCoCo for viewing test coverage😘

### CODE REVIEW

**Comment by MitchellTFord** [comment]
You should add instructions to the "Getting Started" section of the README for executing tests and opening the JaCoCo report.
Also - link the issue this PR resolves

**Comment by LiangZhang21** [comment]
added

File: README.md [hunk 1]
```diff
@@ -28,6 +28,16 @@ cd discord-bot
...
+### Generate & View Code Coverage Report Using JaCoCo
+```shell
+# run the coverage report by either of the following commands in the terminal
+./gradlew test
+#or 
+./gradlew jacocoTestReport
+# You can find the report by navigating to the project directory, 
+# and under build/jacoco/test/html you will find the index.html to view the report
+```
+
```
Comment (MitchellTFord) [inline README.md:40]: This kinda implies that `test` and `jacocoTestReport` do the same thing. I guess they mostly do in our implementation
Maybe try something like this:

---

### Run Unit Tests and Generate Coverage Report Using JaCoCo
```shell
# Run unit tests and generate report
./gradlew test

# Generate report without re-running tests
./gradlew jacocoTestReport
```
You can find the report at `build/jacoco/test/html/index.html`

---

Comment (LiangZhang21) [reply README.md]: 😂👌

Comment (CodeMasterBrandon) [reply README.md]: Not bad code almost there!

**Review by MitchellTFord — APPROVED** [review]
Looks good

**Pull request merged**